File: app/src/main/java/com/example/zhengshang/namepic/SplashActivity.java
package com.example.zhengshang.namepic;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.support.annotation.Nullable;

/**
 * Created by zhengshang on 2017/2/21.
 */

public class SplashActivity extends BaseActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent(SplashActivity.this, MainActivity.class);
                startActivity(intent);
                overridePendingTransition(R.anim.fade_in, R.anim.fade_out);
                finish();
            }
        }, 500);
    }
}

File: app/src/main/java/cn/zhengshang/namepic/activity/SplashActivity.kt
package cn.zhengshang.namepic.activity

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import cn.zhengshang.namepic.R

/**
 * Created by zhengshang on 2017/2/21.
 */
class SplashActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Handler().postDelayed({
            val intent = Intent(this@SplashActivity, MainActivity::class.java)
            startActivity(intent)
            overridePendingTransition(R.anim.fade_in, R.anim.fade_out)
            finish()
        }, 500)
    }
}